src/util/BaseLoginController.js

```diff
@@ -21,6 +21,7 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
   }
 
   return Okta.Controller.extend({
+    className: 'base-login-controller',
```

Comment:
No. there aren't any current cases , so removed the class-name prop from the base class